packages/ckeditor5-dev-env/lib/translations/collect-utils.js

```diff
@@ -97,7 +97,7 @@ const utils = {
 		}
 
 		return [ ...usedContextMap ]
-			.filter( ( [ key, usage ] ) => !usage ) // eslint-disable-line no-unused-vars
```

Comment:
I'd actually revert this change. It's very strange that ESLint doesn't understand that you need only `usage` out of these 2 – just like with callback arguments. Sometimes you define more than you use.